### PR TITLE
Update the fixture of hypervisor in conftest.py

### PR DESCRIPTION
**Pull request by Junefen**
Planning to combine the `hypervisor()` and `hypervisor_create(hypervisor)` to the one `hypervisor()`, and change the scope of  `hypervisor()` to class.

### CODE REVIEW

**Comment by Junefen** [comment]
Test Result
```
# pytest -v --disable-warnings tests/function/test_cli.py 
===================== test session starts =====================
platform linux -- Python 3.9.9, pytest-6.2.5, py-1.10.0, pluggy-1.0.0 -- /usr/bin/python3
cachedir: .pytest_cache
rootdir: /root/workspace/virtwho-test, configfile: pytest.ini
collected 5 items                                                                                                                                

tests/function/test_cli.py::TestCli::test_debug PASSED                [ 20%]
tests/function/test_cli.py::TestCli::test_oneshot PASSED                  [ 40%]
tests/function/test_cli.py::TestCli::test_interval PASSED                 [ 60%]
tests/function/test_cli.py::TestCli::test_print PASSED                     [ 80%]
tests/function/test_cli.py::TestCli::test_config PASSED                   [100%]

================== 5 passed in 469.14s (0:07:49) ===========================
```

**Comment by hkx303** [comment]
LGTM

**Review by hkx303 — APPROVED** [review]
ACK

File: tests/function/test_config.py [hunk 1]
```diff
@@ -346,7 +346,7 @@ def test_configs_in_virtwho_conf(self, virtwho, globalconf, hypervisor_data, ssh
 
     @pytest.mark.tier1
     def test_owner_in_virtwho_conf(self, virtwho, globalconf, hypervisor, hypervisor_data,
```
Comment (hkx303) [inline tests/function/test_config.py:348]: the `hypervisor` also can be deleted, but it doesn't matter

Comment (hkx303) [reply tests/function/test_config.py:348]: And https://github.com/VirtwhoQE/virtwho-test/blob/69c3eb25e87b8f07e414b212d550b79dbc7ccf86/tests/function/test_config.py#L389 is the same

Comment (hkx303) [inline tests/function/test_config.py:348]: Merge this PR for now as we can move it easily in the future, just add a mark

**Pull request merged**